js/src/Ice/ConnectionI.js

```diff
@@ -1488,10 +1531,12 @@ export class ConnectionI {
 }
 
 class OutgoingMessage {
```

Comment:
OutgoingMessage means outgoing request message? request or batch request?

The fields you added don't seem appropriate for an outgoing reply. You're sure OutgoingMessage is not used for replies?
What about outgoing ValidateConnection, outgoing ConnectionClose?

Reply:
> OutgoingMessage means outgoing request message? request or batch request?

It is used for any thing we send with the connection, it can be a request, a response, and validate connection message, a batch request.

> The fields you added don't seem appropriate for an outgoing reply.

we already have the `receivedReply` in the C# version of this.

https://github.com/zeroc-ice/ice/blob/34ce55cb67eee969459b618bfe7e9a5cab91e2c9/csharp/src/Ice/ConnectionI.cs#L2877